src/main.rs

```diff
@@ -75,7 +84,17 @@ fn do_main() -> Result<(), Box<Error>> {
         symbols: RefCell::new(symbols),
         class_hierarchy: class_hierarchy,
     };
-    codegen::generate(&env);
+
+    drop(watcher_loading);
+    {
+        let _watcher_generating = statistics.start(SWType::Generating);
+
+        codegen::generate(&env);
+    }
+
+    drop(watcher_total);
+
+    statistics.print();
```

Comment:
I'd rather we made this optional either via an command-line flag or logging via `info!`.